mlkem/mlkem_native.h

```diff
@@ -56,14 +66,9 @@
  *   MLK_CONFIG_API_PARAMETER_SET or MLK_CONFIG_API_NAMESPACE_PREFIX,
  *   nor include a configuration.
  *
- * # Multi-level builds
+ * - MLK_CONFIG_API_QUALIFIER [optional]
```

Comment:
`MLK_CONFIG_API_QUALIFIER` is not a valid configuration option. Did you want to add that? Or do you want to rename here to the pre-existing option `MLK_CONFIG_EXTERNAL_API_QUALIFIER`